inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/es/ElasticsearchApi.java

```diff
@@ -133,10 +133,10 @@ private List<String> getMappingInfo(List<ElasticsearchFieldInfo> fieldsInfo) {
                     fieldStr.append(",\n          \"format\" : \"")
                             .append(entry.getFormat()).append("\"");
                 }
-            } else if (entry.getType().contains("float")) {
```

Comment:
Suggested change the `entry` to `field`.

Reply:
fixed